test/fzf.vader

```diff
@@ -147,6 +147,23 @@ Execute (fzf#wrap):
   let opts = fzf#wrap({})
   Assert opts.options =~ '^--color=fg:'
 
+Execute (fzf#shellesc with sh):
+  AssertEqual '""', fzf#shellesc('', 'sh')
+  AssertEqual '"\"\""', fzf#shellesc('""', 'sh')
+  AssertEqual '"foobar>"', fzf#shellesc('foobar>', 'sh')
+  AssertEqual '"\\""', fzf#shellesc('\"', 'sh')
```

Comment:
No I was not referring to tmux integration as it's no longer the default (thanks to `--height`).

Maybe I was confused, I thought something like `:call fzf#run({'source': 'echo $PATH'})` wouldn't work if we use builtin `shellescape`, but it is unaffected.